src/ssr-template-helper.js

```diff
@@ -94,6 +94,24 @@ export class SsrTemplateHelper {
         ));
   }
 
+  /**
+   * @param {!Element} container
+   * @param {!JsonObject} data
+   * @return {!Promise}
+   */
+  renderTemplate(container, data) {
```

Comment:
We should also use this in amp-list, right? Sharing more code was the point here.